### DISCUSSION TITLE
Syntax Highlighting not working

### DISCUSSION THREAD

**Discussion by Greeenman999** [coder/code-server#4344] [Q&A]
Web Browser: Google Chrome 94.0.4606.81 (Official Build) (arm64)
Local OS: Mac Os Big Sur 11.3.2
Remote OS: Debian 10
Remote Architecture: amd
code-server --version: 3.12.0 b37ff28

I upgraded code-server and then there was no syntax highlighting anymore.
When I'm accessing directly over the ip adress and port, then its working.
On Firefox it's also working, but on Chrome it doesnt work.

Sometimes this error occures:
<img width="665" alt="Bildschirmfoto_2021-10-12_um_10 27 37" src="https://user-images.githubusercontent.com/81023792/136977582-691b308b-2d20-4289-b1ef-26f7db5f4690.png">
<img width="529" alt="Bildschirmfoto 2021-10-12 um 16 43 52" src="https://user-images.githubusercontent.com/81023792/136978000-6cc71d3d-cbd5-4860-b07e-e7eecb0dadbe.png">

The complete code is not highlighted and gray:
<img width="1438" alt="Bildschirmfoto_2021-10-12_um_10 19 32" src="https://user-images.githubusercontent.com/81023792/136977703-c6b52352-85c1-4e6f-ab9d-97939f18b1c7.png">

Anyone else having this issue?

**Comment by Greeenman999** [comment]
When im accessing over the IP Adress and Port directly then i have Syntax Highlighting, but with nginx not D:

**Reply by Greeenman999** [reply]
Just found out, that its working on Firefox, but still not on Chrome....

**Reply by jsjoeio** [reply]
Hmmm...I'm not sure what this could be exactly. I wonder why the service worker is failing.

What happens if you run `code-server --link` and then access it?

**Reply by cycl0ne** [reply]
Same here. Just installed code-server on a linux VM machine. Works great in chrome, but no syntax highlighting. on Firefox it works.

**Reply by Greeenman999** [reply]
I was able to solve it with a different nginx configuration. What config are you using?

**Reply by cycl0ne** [reply]
Standard nginx that was supplied by the cloud provider. But it seems it is related to my chrome im using here in office. It seems the company has disabled webassembly. If i surf a webassembly test page, it also fails. Edge/Firefox works.
